.github/pull_request_template.md

```diff
@@ -1,27 +1,52 @@
 ## Summary
 
-Basic description of work done. Closes [#issue_no].
+Basic description of work done.
 
-### Preview
+## Preview
 
 [Link to Preview]()
 
 <!--
 ⚠️ Significant visual changes require submitting previous design to wayback machine.
 -->
 
-### Solution
+## Solution
 
-### How To Test
+Provide a summary of the solution this PR offers.
+
+<!--
+It can be helpful if we understand:
+1. What the solution is,
+2. Why this approach was chosen,
+3. How you implemented the change, and
+4. Possible limitations of this approach and alternate solution paths.
+-->
+
+
+## How To Test
 
 1. First Step
 2. Second Step
 3. Third Step
 
----
+<!--
+For PRs that include dependency updates, uncomment this section and
+include a list of the changed dependencies and version numbers.
+-->
+
+<!--
+## Dependency updates
 
-### Dev Checklist
+| Dependency name              | Previous version | New version |
+| ---------------------------- | :--------------: | :---------: |
+| [Updated dependency example] |     [1.0.0]      |   [1.0.1]   |
+| [New dependency example]     |        --        |   [3.0.1]   |
+| [Removed dependency example] |     [2.10.2]     |     --      |
+-->
```

Comment:
I like this addition, good call.

Reply:
Thanks, got it from the guidance @amyleadem created in USWDS.